ui/src/containers/organisations.list/billingaccounts/subscriptions/columns.tsx

```diff
@@ -54,14 +51,23 @@ export const getColumns: (
       meta: {
         headerFilter: false,
       },
-      cell: (info) =>
-        new Date(info.getValue() as Date).toLocaleString("en", {
-          month: "long",
-          day: "numeric",
-          year: "numeric",
-        }),
+      cell: (info) => getFormattedDateString(info.getValue()),
 
       footer: (props) => props.column.id,
     },
+    {
+      header: "Cancellation date",
```

Comment:
Let's stick to `Cancelation` with a single `L`.
I had mistakenly suggested double `L` but that is not as per American English (which is what we have used at most places)